components/fieldlabel/metadata/form.yml

```diff
@@ -7,25 +7,25 @@ examples:
     markup: |
       <form class="spectrum-Form">
         <div class="spectrum-Form-item">
-          <label for="fieldLabelExample-lifestory" class="spectrum-FieldLabel spectrum-FieldLabel--sizeM spectrum-Form-itemLabel spectrum-FieldLabel--left">Company Title</label>
+          <label for="fieldLabelExample-company-1" class="spectrum-FieldLabel spectrum-FieldLabel--sizeM spectrum-Form-itemLabel spectrum-FieldLabel--left">Company Title</label>
           <div class="spectrum-Form-itemField">
             <div class="spectrum-Textfield spectrum-Textfield--multiline">
-              <textarea id="fieldLabelExample-lifestory" placeholder="Enter your life story" name="field" class="spectrum-Textfield-input"></textarea>
+              <textarea id="fieldLabelExample-company-1" name="field" class="spectrum-Textfield-input"></textarea>
             </div>
           </div>
         </div>
         <div class="spectrum-Form-item">
-          <label class="spectrum-FieldLabel spectrum-FieldLabel--sizeM spectrum-Form-itemLabel spectrum-FieldLabel--left" for="fieldLabelExample-emailaddress">Email Address</label>
+          <label class="spectrum-FieldLabel spectrum-FieldLabel--sizeM spectrum-Form-itemLabel spectrum-FieldLabel--left" for="fieldLabelExample-emailaddress-1">Email Address</label>
           <div class="spectrum-Form-itemField">
             <div class="spectrum-Textfield">
-              <input class="spectrum-Textfield-input" aria-invalid="false" type="text" placeholder="Enter your e-mail address" id="fieldLabelExample-emailaddress">
+              <input class="spectrum-Textfield-input" aria-invalid="false" type="text" id="fieldLabelExample-emailaddress-1">
             </div>
           </div>
         </div>
         <div class="spectrum-Form-item">
-          <label class="spectrum-FieldLabel spectrum-FieldLabel--sizeM spectrum-Form-itemLabel spectrum-FieldLabel--left" for="spectrum-textinput-instance">Country</label>
+          <div class="spectrum-FieldLabel spectrum-FieldLabel--sizeM spectrum-Form-itemLabel spectrum-FieldLabel--left">Country</div>
```

Comment:
Wondering if this should still be `label` and if we need a `for` and `id` pair on these for the example to be accessible?

Reply:
There's no actual form field in the picker (though it feels like there should be a hidden select there!), and the label element `for` is [only for certain "labelable" elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#for) , which is why I changed this after seeing a warning, to match the picker docs which use a div.

Reply:
Ahh, gotcha. Thanks for explaining!

Reply:
👍 It does look like the `button` is considered "labelable" by `for` on `label`; on the country fields I've set this back to labels and have connected the two with IDs. I had based this on the picker docs markup, but maybe those should change as well!